GetMessage/userPreferenceChecker/userPreferencesCheckerFactory.ts

```diff
@@ -105,7 +105,10 @@ export const isAppVersionHandlingReadAuthorization = (
   minAppVersionHandlingReadAuth: Semver,
   currentAppVersion: Semver
 ): boolean => {
-  const currentAppVersionWithoutBuild = currentAppVersion.slice(0, 5);
+  const currentAppVersionWithoutBuild = currentAppVersion
```

Comment:
Add a comment that explain why we are excluding the build number. Even a function doc (it's a private function but is exported so documentation is not a wast of time :D)?

Reply:
Right, done in [5e411b9](https://github.com/pagopa/io-functions-services/pull/211/commits/5e411b9a1d3b66bded7563733c2ab8d21530e914)